From: Sergei Zviagintsev <sergei@s15v.net>
Date: Wed, 10 Jun 2015 00:00:02 +0300
Subject: [PATCH] Documentation/kdbus: use parentheses uniformly in
 KDBUS_ITEM_FOREACH macro

Enclose all arguments into parentheses to stay consistent across the
whole macro.

Signed-off-by: Sergei Zviagintsev <sergei@s15v.net>
Reviewed-by: David Herrmann <dh.herrmann@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 Documentation/kdbus/kdbus.item.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/kdbus/kdbus.item.xml b/Documentation/kdbus/kdbus.item.xml
index b0eeeef995af..ee09dfa443b8 100644
--- a/Documentation/kdbus/kdbus.item.xml
+++ b/Documentation/kdbus/kdbus.item.xml
@@ -72,10 +72,10 @@
     (typeof(item))((uint8_t *)(item) + KDBUS_ALIGN8((item)->size))
 
 #define KDBUS_ITEM_FOREACH(item, head, first)                      \
-    for (item = (head)->first;                                     \
+    for ((item) = (head)->first;                                   \
          ((uint8_t *)(item) < (uint8_t *)(head) + (head)->size) && \
           ((uint8_t *)(item) >= (uint8_t *)(head));                \
-         item = KDBUS_ITEM_NEXT(item))
+         (item) = KDBUS_ITEM_NEXT(item))
       ]]></programlisting>
     </refsect2>
   </refsect1>
